src/js/player.js

```diff
@@ -2672,6 +2673,62 @@ class Player extends Component {
     this.trigger('exitFullWindow');
   }
 
+  /**
+   * Handle a keydown event from any Component of this player which is not
+   *  handled by that Component to support player-wide hotkeys.
+   *
+   * @param {EventTarget~Event} event
+   *        The `keydown` event that was passed from a Component.
+   */
+  handleKeyPress(event) {
+
+    if (this.options_.hotkeys !== false) {
+
+      if (typeof this.options_.hotkeys === 'function') {
+
+        this.options_.hotkeys(event);
+
+      // Trigger the ControlBar equivalent for each key, if possible
+      } else if (keycode.isEventKey(event, 'f')) {
+        // "f" = toggle fullscreen
+
+        event.preventDefault();
+
+        const cb = this.player_.controlBar;
```

Comment:
Should we also define, and check for this at the top of the function? That would prevent some of the code duplication in each if statement.